cassava-megaparsec.cabal

```diff
@@ -1,7 +1,7 @@
+cabal-version:        3.0
 name:                 cassava-megaparsec
-version:              2.0.4
-cabal-version:        1.18
-tested-with:          GHC==7.10.3, GHC==8.0.2, GHC==8.2.2, GHC==8.4.3, GHC==8.6.1
+version:              2.1.0
+tested-with:          GHC==8.2.2, GHC==8.4.3, GHC==8.6.1, GHC==8.10, GHC==9.0
```

Comment:
I suggest to put the latest minor releases of all GHC versions you support here.  In the [hackage revision](https://hackage.haskell.org/package/cassava-megaparsec-2.0.4/cassava-megaparsec.cabal), I put: 
```
tested-with:
  GHC == 9.4.4
  GHC == 9.2.5
  GHC == 9.0.2
  GHC == 8.10.7
  GHC == 8.8.4
  GHC == 8.6.5
  GHC == 8.4.4
  GHC == 8.2.2
```